tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": "@sindresorhus/tsconfig",
+  "target": "ES2018",
```

Comment:
```suggestion
```

Except if I'm missing something, I think it can be removed since the target is higher in https://github.com/sindresorhus/tsconfig/blob/main/tsconfig.json#L7

Reply:
Yes, you are right, I didn't notice, I thought it was less.